data-prepper-api/src/main/java/org/opensearch/dataprepper/model/PipelineIf.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.model;
+
+import org.opensearch.dataprepper.model.source.Source;
+
+public interface PipelineIf {
```

Comment:
What does `PipelineIf` mean?

Reply:
I guess I could just use `Pipeline` here instead of introducing a new interface